src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure implements AreaCalculetion {
+    private double sideB;
+    private double sideA;
+
+    public Rectangle(double sideA, double sideB, String color, String name) {
+        this.sideA = sideA;
+        this.sideB = sideB;
+    }
+
+    public double getSideA() {
+        return sideA;
+    }
+
+    public double getSideB() {
+        return sideB;
+    }
+
+    @Override
+    public double getArea() {
+        return getSideA() * getSideB();

```

Comment:
Instead of calling the getters inside the same class, you can use the fields `sideA` and `sideB` directly. This can improve performance as it avoids the overhead of method calls.